### PR TITLE
Update deps

### PR DESCRIPTION
**Pull request by grahamscott**
@spmason @bbmaj

### CODE REVIEW

**Review by spmason — APPROVED** [review]
LGTM

File: hermes.js [hunk 1]
```diff
@@ -13,9 +13,8 @@
  */
 
 // IIFE to protect global scope.
+'use strict';
```
Comment (qubyte) [inline hermes.js]: Probably fine for most needs, but if anyone out there is using this in a script tag, then putting this outside the IIFE might affect other code.

**Comment by grahamscott** [comment]
@qubyte good point - I've disabled the lint rule for this file instead 👍

**Review by qubyte — APPROVED** [review]
Looks good to me too.

**Pull request merged**